src/Pages/Account/index.jsx

```diff
@@ -0,0 +1,22 @@
+import UserEventData from '../../Components/UserEventData'
+import useAuth from '../../Context'
+
+export const Account = () => {
+  const { userSpotifyID, logout } = useAuth()
+
+  if (userSpotifyID === '') {
+    return null
```

Comment:
so this is fine for now but maybe we want a ticket in the future to add some loading state to useAuth so that we know if the user is being loaded or just does not exist

Reply:
added ticket to trello for this